app/models/metric/capture.rb

```diff
@@ -43,8 +43,7 @@ def self.capture_threshold(target)
   # Capture entry points
   #
 
-  def self.perf_capture_health_check(zone = nil)
-    zone ||= MiqServer.my_server.zone(true)
+  def self.perf_capture_health_check(zone)
```

Comment:
I feel like this method should be private as well.  We don't call it from anywhere else, but perf_capture_timer.